api/v01/vrp.rb

```diff
@@ -162,6 +162,32 @@ class Vrp < APIBase
           end
         end
 
+        resource :validate do
+          desc 'Validate VRP problem', {
+            nickname: 'submit_vrp',
```

Comment:
this is the same nickname with the POST, is it voluntary (wouldn't it lead to a clash)? If not, we can have `validate_vrp` as the nickname
```suggestion
            nickname: 'validate_vrp',
```